src/payee/handler.js

```diff
@@ -251,16 +259,22 @@ exports.postQuotes = function (req, h) {
           'FSPIOP-Signature': `${JSON.stringify(fspiopSignature)}`,
           'FSPIOP-HTTP-Method': 'PUT',
           'FSPIOP-URI': `/quotes/${quotesRequest.quoteId}`,
-          traceparent: req.headers.traceparent ? req.headers.traceparent : undefined,
-          tracestate: req.headers.tracestate ? req.headers.tracestate : undefined
+          traceparent: req.headers.traceparent ? req.headers.traceparent : '',
+          tracestate: req.headers.tracestate ? req.headers.tracestate : ''
         },
-        rejectUnauthorized: false,
-        body: JSON.stringify(quotesResponse)
+        transformRequest: [(data, headers) => {
+          delete headers.common.Accept
+          return data
+        }],
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false
+        }),
+        data: JSON.stringify(quotesResponse)
       }
       Logger.info((new Date().toISOString()), 'Executing PUT', url)
-      const res = await fetch(url, opts)
+      const res = await request(url, opts)
```

Comment:
I dont believe that this would work. axios requires the url to be sent in the opts. therefore your call should look like 
```
const opts = {
    url: '',
    method: 'PUT',
    headers: [],
    ....
}
const res = await request(opts)
```